go.sum

```diff
@@ -432,8 +432,8 @@ google.golang.org/grpc v1.21.0/go.mod h1:oYelfM1adQP15Ek0mdvEgi9Df8B9CZIaU1084ij
 google.golang.org/grpc v1.23.0/go.mod h1:Y5yQAOtifL1yxbo5wqy6BxZv8vAUGQwXBOALyacEbxg=
 google.golang.org/grpc v1.25.1/go.mod h1:c3i+UQWmh7LiEpx4sFZnkU36qjEYZ0imhYfXVyQciAY=
 google.golang.org/grpc v1.27.0/go.mod h1:qbnxyOmOxrQa7FizSgH+ReBfzJrCY1pSN7KXBS8abTk=
-google.golang.org/grpc v1.30.1 h1:oJTcovwKSu7V3TaBKd0/AXOuJVHjTdGTutbMHIOgVEQ=
-google.golang.org/grpc v1.30.1/go.mod h1:N36X2cJ7JwdamYAgDz+s+rVMFjt3numwzf/HckM8pak=
+google.golang.org/grpc v1.33.0-dev h1:c0EY3sGPLj50wEdGQDpiS3zvk/zdduzrAkJTfa9ocjY=
```

Comment:
Is `dev` an unstable version?

Reply:
@AskAlexSharov I think only you has the right package and libs versions, I've tried to install versions from master and run `devtools` many times and always get `packageVersion7`.
Could you checkout to this branch and regenerate files from proto files and then push results to the branch?

Reply:
So. 
1. "Is dev an unstable version?" - 1.31 their current release, 1.32-dev their next release, 1.33-dev their next-next release. 
2. Go code generates by google.golang.org/grpc/cmd/protoc-gen-go-grpc - it means we looking for version reproducibility of this particular tool. 
3. When I run `make devtools` in master - it added line `google.golang.org/grpc/cmd/protoc-gen-go-grpc` to go.mod file (I thought that I added this live with exact version already). Reason why it's not there: `go mod tidy` removes it because our source code doesn't use it - it's true - we don't use this package from source code, but we use it to generate go code. 
4. So, probably we can trick `go mod tidy` - add dummy package where I will use all our binary dependencies. But nope, we can do it only if such dep has non-main package: https://github.com/golang/go/issues/32504
5. But I found how to trick it anyway: https://github.com/ledgerwatch/turbo-geth/pull/1037 
6. I pointed `google.golang.org/grpc/cmd/protoc-gen-go-grpc` to commit of `v1.30.1` tag